portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.data;
+
+import java.time.LocalDateTime;
+
+/** Represents a comment with related details. */
+public final class Comment {
+    private final String name;
+    private final String comment;
```

Comment:
Nit: for clarity, can also be "content", so we can have something like comment.getContent() instead of comment.getComment()?

Reply:
Fixed